### ISSUE TITLE
Support filters with complex transform

### ISSUE THREAD

**Issue by RazrFalcon** [RazrFalcon/resvg#254]

**Comment by LaurenzV** [comment]
Isn't this already supported? Or what exactly is meant by complex transforms?

**Comment by RazrFalcon** [comment]
Nope. I still have no idea how to implement it. Just grep filter tests for "complex transform". They are all failing.
